### ISSUE TITLE
FormBuilderTextBox should support context menu as default

### ISSUE THREAD

**Issue by yfakariya** [flutter-form-builder-ecosystem/flutter_form_builder#1194]
## Environment

**Package version:** 7.8.0

## Description

**What you'd like to happen:**

`FormBuilderTextBox ` supports default context menu out of box.

Currently, default value for `contextMenuBuilder` parameter of `FormBuilderTextBox` is [not specified](https://github.com/flutter-form-builder-ecosystem/flutter_form_builder/blob/4c83c5c9f8d235d29db2045620b6d656814ebc13/lib/src/fields/form_builder_text_field.dart#L352), so no context menu is shown as default.
In contrast, native `TextFormField` [specifies](https://github.com/flutter/flutter/blob/1d17caed66641b55a8e8faadf212398f746f8861/packages/flutter/lib/src/material/text_form_field.dart#L156) [default implementation](https://github.com/flutter/flutter/blob/1d17caed66641b55a8e8faadf212398f746f8861/packages/flutter/lib/src/material/text_form_field.dart#L250-L254) using `AdaptiveTextSelectionToolbar.editableText`.
It is great if `FormBuilderTextField` also support default context menu.

For your reference, the default context menu enables pasting from clipboard, selecting all inputs if there are any inputs, and cutting and copying seleted texts. I think these functions are "must-have" for UX.

**Comment by tsinis** [comment]
@yfakariya as a solution for now you can use (in your `FormBuilderTextField`):

```dart
contextMenuBuilder: (_, editableTextState) =>
    AdaptiveTextSelectionToolbar.editableText(
  editableTextState: editableTextState,
),
```

**Comment by deandreamatias** [comment]
Solved by #1212
